deps/src/qt-bluetooth/qlowenergyservice.cpp

```diff
@@ -699,7 +699,7 @@ void QLowEnergyService::writeCharacteristic(
         const QByteArray &newValue, QLowEnergyService::WriteMode mode)
 {
     //TODO check behavior when writing to WriteSigned characteristic
-    Q_D(QLowEnergyService);
+   Q_D(QLowEnergyService);
```

Comment:
Generally, we don't want to see any files committed that aren't relevant. Please fix this and others in the PR.

Reply:
@dioptre please checkout to the latest commit I have updated my code with the latest version.